pom.xml

```diff
@@ -353,6 +358,38 @@
             </execution>
           </executions>
         </plugin>
+        <!-- Spotless plugin: keeps the code formatted following the google-java-styleguide -->
+        <plugin>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>2.40.0</version>
+          <configuration>
+            <formats>
+              <format>
+                <includes>
+                  <include>*.md</include>
+                  <include>.gitignore</include>
+                </includes>
+                <trimTrailingWhitespace/>
+                <endWithNewline/>
+                <indent>
+                  <tabs>true</tabs>
+                  <spacesPerTab>4</spacesPerTab>
+                </indent>
+              </format>
+            </formats>
+            <java>
+              <googleJavaFormat>
+                <style>GOOGLE</style>
+                <reflowLongStrings>true</reflowLongStrings>
+                <formatJavadoc>true</formatJavadoc>
+              </googleJavaFormat>
+              <importOrder />             <!-- Checks imports order -->
+              <removeUnusedImports />     <!-- Removes unused imports -->
+              <formatAnnotations />       <!-- Formats the annotations -->
```

Comment:
...except here, where if I'm reading the documentation correctly this is specifically about putting type annotations immediately before the type, rather than on the preceding line.

Reply:
I’m not sure if I understood correctly, I removed the comments on `<importOrder />` and `<removeUnusedImports />`

Reply:
I'm suggesting that the comment could read something like `<!-- Puts type annotations immediately before types. -->`. I'm not even sure that's necessary, because google-java-format may do it anyway, but it's what the documentation [says](https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#formatAnnotations).